docs/content/en/docs/clustermgmt/cluster-upgrades/management-components-upgrade.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Upgrade management components"
+linkTitle: "Upgrade management components"
+weight: 21
+aliases:
+    /docs/tasks/cluster/cluster-upgrades/management-components-upgrade/
+date: 2024-02-21
+description: >
+  How to perform management components upgrade for management clusters
+---
+
+>**_NOTE:_** The `upgrade management-components` sub-command is added in `EKS Anywhere` version `v0.19.0` for all providers. Management components upgrades can only be done through the `eksctl anywhere` cli, not through the Kubernetes API.
```

Comment:
The `eksctl anywhere upgrade management-components` subcommand was added in EKS Anywhere version `v0.19.0` for all providers. Management component upgrades can only be done through the `eksctl` CLI, not through the Kubernetes API.